### PR TITLE
Add netperf histogram support

### PR DESCRIPTION
**Pull request by ehankland**

### CODE REVIEW

File: perfkitbenchmarker/linux_packages/netperf.py [hunk 1]
```diff
@@ -15,13 +15,25 @@
...
 """Module containing netperf installation and cleanup functions."""
 
+import re
+
+from perfkitbenchmarker import flags
 from perfkitbenchmarker import vm_util
 
+flags.DEFINE_integer(
+    'netperf_histogram_buckets', 100,
+    'The number of buckets per row in the internal histogram that netperf '
```
Comment (yuyantingzero) [inline perfkitbenchmarker/linux_packages/netperf.py]: Maybe have a more details about row. "Netperf internal histogram" is not very obvious.

File: perfkitbenchmarker/linux_packages/netperf.py [hunk 2]
```diff
@@ -42,3 +61,47 @@ def YumInstall(vm):
...
+  # UNIT_MSEC     :    0:   24:   57:   40:    5:    2:    0:    0:    0:    0
+  # TEN_MSEC      :    0:    0:    0:    0:    0:    0:    0:    0:    0:    0
+  # HUNDRED_MSEC  :    0:    0:    0:    0:    0:    0:    0:    0:    0:    0
+  # UNIT_SEC      :    0:    0:    0:    0:    0:    0:    0:    0:    0:    0
+  # TEN_SEC       :    0:    0:    0:    0:    0:    0:    0:    0:    0:    0
+  # >100_SECS: 0
+  # HIST_TOTAL:      444658
+  match = re.search('(UNIT_USEC.*?)>100_SECS', netperf_stdout, re.S)
+  if not match:
+    return None
```
Comment (yuyantingzero) [inline perfkitbenchmarker/linux_packages/netperf.py]: Add a warning and log raw results if doesn't match?

File: perfkitbenchmarker/linux_packages/netperf.py [hunk 3]
```diff
@@ -42,3 +61,47 @@ def YumInstall(vm):
...
+  # TEN_USEC      :    0:    0:    0:    0:    0:    0:    0:    0:    0:    0
+  # HUNDRED_USEC  :    0: 433684: 9696:  872:  140:   56:   27:   28:   17:   10
+  # UNIT_MSEC     :    0:   24:   57:   40:    5:    2:    0:    0:    0:    0
+  # TEN_MSEC      :    0:    0:    0:    0:    0:    0:    0:    0:    0:    0
+  # HUNDRED_MSEC  :    0:    0:    0:    0:    0:    0:    0:    0:    0:    0
+  # UNIT_SEC      :    0:    0:    0:    0:    0:    0:    0:    0:    0:    0
+  # TEN_SEC       :    0:    0:    0:    0:    0:    0:    0:    0:    0:    0
+  # >100_SECS: 0
+  # HIST_TOTAL:      444658
+  match = re.search('(UNIT_USEC.*?)>100_SECS', netperf_stdout, re.S)
```
Comment (yuyantingzero) [inline perfkitbenchmarker/linux_packages/netperf.py]: Use https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/regex_util.py?

File: perfkitbenchmarker/linux_packages/netperf.py (hunk 1 above)
Comment (ehankland) [inline perfkitbenchmarker/linux_packages/netperf.py]: Done. Please check to make sure the new description is understandable

File: perfkitbenchmarker/linux_packages/netperf.py (hunk 2 above)
Comment (ehankland) [inline perfkitbenchmarker/linux_packages/netperf.py]: Done - Moved to regex_util which does takes care of this for me

File: perfkitbenchmarker/linux_packages/netperf.py (hunk 3 above)
Comment (ehankland) [inline perfkitbenchmarker/linux_packages/netperf.py]: Done

**Comment by ehankland** [comment]
@yuyantingzero PTAL

**Comment by yuyantingzero** [comment]
LGTM

**Pull request merged**